src/audit-logs/audit-logs.ts

```diff
@@ -0,0 +1,16 @@
+import { WorkOS } from '../workos';
+import { AuditLogEventOptions } from './interfaces';
+
+export class AuditLogs {
+  constructor(private readonly workos: WorkOS) {}
+
+  async createEvent(
```

Comment:
It'll be important to allow customers to pass an `idempotencyKey` like we do in the old API to prevent accidental creation of duplicate events.